src/PluginsManager/Microsoft.Performance.Toolkit.PluginsManager.Core/Utils/HashUtils.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Performance.SDK;
+
+namespace Microsoft.Performance.Toolkit.PluginsManager.Core.Utils
+{
+    /// <summary>
+    ///     Provides static methods for calculating hash.
+    /// </summary>
+    public static class HashUtils
```

Comment:
Does this make sense to move somewhere closer to the runtime?

Reply:
To where specifically? Something to be passed down to the SDK by the runtime?

Reply:
We have HashCodeUtils, so probably place this file next to that